### ISSUE TITLE
Add public protocol buffer API/message definitions to release tarball [phab:D505]

### ISSUE THREAD

**Issue by kythe** [google/kythe#743]
_Created by **schroederc** at 2015-09-08 19:25:08_: 

This adds the following files to the release tarball:

```
/proto
├── analysis.proto
├── cxx.proto
├── filetree.proto
├── java.proto
├── storage.proto
└── xref.proto
```

**Comment by kythe** [comment]
_**jdennett** wrote at 2015-09-08 20:14:33_:

Could you describe the effect of this change in more depth (in the change description and the release notes)?

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-09-08 20:15:29_:

>>! In (earlier comment), jdennett wrote:
> Could you describe the effect of this change in more depth (in the change description and the release notes)?

Not really, this just adds the protos to the release archive.

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-09-10 18:18:00_:

Ping

**Comment by kythe** [comment]
_**jdennett** wrote at 2015-09-10 18:46:04_:

Could you say "tarball" or ".tar.gz" to be more specific than the term "archive"?

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-09-10 18:47:55_:

>>! In (earlier comment), jdennett wrote:
> Could you say "tarball" or ".tar.gz" to be more specific than the term "archive"?

Done

**Comment by kythe** [comment]
_**jdennett** commented inline (context lost during migration) at 2015-09-10 19:40:34_:

Make this roughly match the patch description, maybe just s/Release/Add/.

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-09-10 19:53:15_:

Word

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-09-10 19:54:11_:

Done

**Comment by kythe** [comment]
_**jdennett** accepted this revision at 2015-09-10 21:57:47_

**Comment by kythe** [comment]
_**schroederc** closed via commit 4ec20321269d648faffcc7414ad3bc60b77b57bf at 2015-09-10 21:58:11_
